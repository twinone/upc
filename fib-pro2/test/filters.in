+ 10:30 hi #true #unused
+ 10:31 bye #asd
+ 10:32 ok #asd #twinone_rocks
? #true
? #false
? (#true.#true)
? (#true.#false)
? (#false,#true)
? (#false,#false)
? (#true,(#false,(#false,(#false,#true))))
? (#false.((#true.#false),(#false,#true)))
? ((#true.#unused),(#asd.#twinone_rocks))
sortir

######################################################
#
# Author: Twinone
#
# Tests tags for the following:
#
# A filter that matches a task
#
# A filter that doesn't match any task
#
# Simple AND filter: True
#
# Simple AND filter: False
#
# Simple OR filter: True
#
# Simple OR filter: False
#
# OR skip test (true or x doesn't evaluate x)
#
# AND skip test (false or x doesn't evaluate x)
#
# OR of ANDS matching multiple tasks
#
######################################################

