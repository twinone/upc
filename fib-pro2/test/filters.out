
+ 10:30 hi #true #unused

+ 10:31 bye #asd

+ 10:32 ok #asd #twinone_rocks

? #true
1 hi 20.04.15 10:30 #true #unused

? #false

? (#true.#true)
1 hi 20.04.15 10:30 #true #unused

? (#true.#false)

? (#false,#true)
1 hi 20.04.15 10:30 #true #unused

? (#false,#false)

? (#true,(#false,(#false,(#false,#true))))
1 hi 20.04.15 10:30 #true #unused

? (#false.((#true.#false),(#false,#true)))

? ((#true.#unused),(#asd.#twinone_rocks))
1 hi 20.04.15 10:30 #true #unused
2 ok 20.04.15 10:32 #asd #twinone_rocks

sortir
